lib/Core/Executor.cpp

```diff
@@ -1102,8 +1104,16 @@ ref<klee::ConstantExpr> Executor::evalConstant(const Constant *c) {
       }
       ref<Expr> res = ConcatExpr::createN(kids.size(), kids.data());
       return cast<ConstantExpr>(res);
+    } else if (const ConstantVector *cv = dyn_cast<ConstantVector>(c)) {
+      ref<Expr> *kids = new ref<Expr>[ cv->getNumOperands() ];
```

Comment:
Yes this seems like a good idea. The reason the code is written this way is that I inherited this code from KLEE-FP and didn't bother to change it.